scripts/filter_type.js

```diff
@@ -0,0 +1,82 @@
+//used for the filter type to be determined once x_axis and y_axis are selected
+
+$(document).ready(function() {
+    var plots = [];
+
+    $("#y-axis-search, #x-axis-search").change(function() {
+        $("#filter-type").empty().append('<option disabled selected value> -Filter- </option>')
+        $.getJSON(json_url, function(plots) {
+
+            var y_axis = $("#y-axis-search");
+            var x_axis = $("#x-axis-search");
+            var dist_options = [];
```

Comment:
be consistent with your naming. For javascript, `camelCase` your names, not `kebab-case`

Reply:
fixed